.github/ISSUE_TEMPLATE/task.yml

```diff
@@ -0,0 +1,37 @@
+name: "Task Report:  Groupware"
+description: "Create new meta task"
+title: "[Task]: "
```

Comment:
```suggestion
title: ""
```

Reply:
it doesnt allow empty string.

Reply:
then drop the line

Reply:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#top-level-syntax